### PR TITLE
Try configuring dockerd with systemd drop-in

### PR DESCRIPTION
**Pull request by jscook2345**
This is a test

### CODE REVIEW

File: .travis/before_install [hunk 1]
```diff
@@ -5,12 +5,13 @@ if [ -z "$OS_TYPE" ]; then
     exit
 fi
 
-# run docker as shown at
-#  https://djw8605.github.io/2016/05/03/building-centos-packages-on-travisci/
+sudo mkdir -p /etc/systemd/system/docker.service.d
+echo "[Service]"  | sudo tee -a /etc/systemd/system/docker.service.d/docker.conf >/dev/null
```
Comment (sashayakovtseva) [inline .travis/before_install:9]: Maybe you can use this syntax here?
```
sudo sh -c 'cat > /etc/systemd/system/docker.service.d/docker.conf <<EOF
[Service]
ExecStart=/usr/bin/dockerd -H tcp://127.0.0.1:2375 -H unix:///var/run/docker.sock -s devicemapper
EOF'
```

Just a suggestion.

File: .travis/before_install [hunk 2]
```diff
@@ -5,12 +5,13 @@ if [ -z "$OS_TYPE" ]; then
...
+echo "ExecStart=" | sudo tee -a /etc/systemd/system/docker.service.d/docker.conf >/dev/null
```
Comment (sashayakovtseva) [inline .travis/before_install:10]: Will this result in the following:
```
[Service]
ExecStart=
ExecStart=/usr/bin/dockerd -H tcp://127.0.0.1:2375 -H unix:///var/run/docker.sock -s devicemapper
```

Double ExecStart is intended?

**Review by sashayakovtseva — CHANGES_REQUESTED** [review]
Need to clarify double `ExecStart` in service

**Comment by jscook2345** [comment]
OOps. This should of been closed. Perhaps your comments could apply to the current code, however?

**Pull request closed without merging**